spec/controllers/posts_controller_spec.rb

```diff
@@ -1718,26 +1722,6 @@
         expect(new_author.joined_at).to be_nil
       end
 
-      it "adds new users to board cameos" do
-        board = create(:board)
-        board.coauthors << create(:user)
-        expect(board.open_to_anyone?).to be(false)
-        expect(board.reload.cameos.count).to eq(0)
-        cameo_user = create(:user)
-        post = create(:post, board: board, user: board.creator)
-        login_as(post.user)
-        expect {
-          put :update, params: {
-            id: post.id,
-            post: {
-              unjoined_author_ids: [cameo_user.id]
-            }
-          }
-        }.to change { PostAuthor.count }.by(1)
-        expect(board.cameos.count).to eq(1)
-        expect(board.cameos).to eq([cameo_user])
-      end
-
```

Comment:
What's up with this being removed? I think we want to make sure it does this on both create and update?

Reply:
Yep, checked, line 1796 has the same test for update, it was in fact a duplicate